packages/dd-trace/test/ci-visibility/dynamic-instrumentation/dynamic-instrumentation.spec.js

```diff
@@ -0,0 +1,48 @@
+'use strict'
+
+require('../../../../dd-trace/test/setup/tap')
+
+const { fork } = require('child_process')
+const path = require('path')
+
+const { assert } = require('chai')
+
+describe('test visibility with dynamic instrumentation', () => {
+  let childProcess
+
+  afterEach(() => {
+    if (childProcess) {
+      childProcess.kill()
+    }
+  })
+
+  it('can grab local variables', (done) => {
+    // Dynamic Instrumentation - Test Visibility not currently supported for windows
+    if (process.platform === 'win32') {
+      done()
+      return
+    }
```

Comment:
Why not move this whole block to the root of the spec file before the first call to `describe`?

Reply:
of course, that's better, thanks

Reply:
done in https://github.com/DataDog/dd-trace-js/pull/4826/commits/ec347a265482e0de365a16525c6d55a0d3ee05b4